### PR TITLE
[2302] PE - V1.0 (Prod, FF, MC) Intune Graph Doc Publishing

### PR DESCRIPTION
**Pull request by jaiprakashmb**
2302 Intune v1 Doc Release - Updated Changelog

### CODE REVIEW

**Comment by Lauragra** [comment]
@jaiprakashmb , this PR is missing a changelog update. Please add that to the PR. Thanks.

**Comment by jaiprakashmb** [comment]
> Latest change log has been updated.

File: changelog/Microsoft.Intune.json [hunk 1]
```diff
@@ -1,5 +1,39 @@
 {
   "changelog": [
+    {
```
Comment (Lauragra) [inline changelog/Microsoft.Intune.json:3]: @jaiprakashmb , this ChangeList record doesn't match the updates in the PR. Please verify that you are using the correct ChangeList.

Comment (jaiprakashmb) [reply changelog/Microsoft.Intune.json:3]: Changelog is correct, I have removed extra files from this PR. Thank!

File: changelog/Microsoft.Intune.json [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **certificateStatus** enumeration type.",
+          "Target": "certificateStatus"
+        },
+        {
+          "Id": "ee873dfb-9f75-423e-8f1c-93aca05e6edb",
+          "ApiChange": "Resource",
+          "ChangedApiName": "enterpriseCodeSigningCertificate",
+          "ChangeType": "Addition",
+          "Description": "Added the [enterpriseCodeSigningCertificate](https://learn.microsoft.com/graph/api/resources/intune-enterpriseCodeSigningCertificate?view=graph-rest-1.0) resource",
```
Comment (Lauragra) [inline changelog/Microsoft.Intune.json:18]: I won't block on this but if in the future, we can add periods to the descriptions, that will be helpful.

**Review by Lauragra — APPROVED** [review]

**Comment by Lauragra** [comment]
@jaiprakashmb , I still can't merge this PR because it's failing the CRLF validation. Please see the instructions to fix the issue with the line endings.

**Comment by jaiprakashmb** [comment]
> @jaiprakashmb , I still can't merge this PR because it's failing the CRLF validation. Please see the instructions to fix the issue with the line endings.

Looks like there was a portal update done by you (2abb4346b6 Updated descriptions), this had introduced CRLF issues. I have fixed it, can you please review it and merge it?

**Review by Lauragra — APPROVED** [review]

**Pull request merged**